bases/bot_detector/api_public/src/core/database/database.py

```diff
@@ -3,15 +3,26 @@
 from sqlalchemy.orm import declarative_base, sessionmaker
 
 # Create an async SQLAlchemy engine
-engine = create_async_engine(
-    settings.DATABASE_URL,
-    pool_pre_ping=True,
-    pool_size=10,
-    max_overflow=90,
-    pool_timeout=settings.POOL_TIMEOUT,
-    pool_recycle=settings.POOL_RECYCLE,
-    # echo=(settings.ENV != "PRD"),
-)
+from sqlalchemy.pool import StaticPool  # add at the top with your imports
+
+# Create an async SQLAlchemy engine
+if settings.DATABASE_URL.startswith("sqlite"):

```

Comment:
we should actually be able to mock the interfaces used for the database

Reply:
pulled this back as well.